src/xdpinstaller/Product.wxs

```diff
@@ -0,0 +1,97 @@
+		<MajorUpgrade AllowSameVersionUpgrades="yes"
+					  Disallow="yes" DisallowUpgradeErrorMessage="An older version of [ProductName] is already installed. Please remove it first."
+					  AllowDowngrades="no" DowngradeErrorMessage="A newer version of [ProductName] is already installed. Please remove it first." Schedule="afterInstallFinalize" />
+		<MediaTemplate EmbedCab="yes" />
+
+		<!-- Define global properties -->
+		<PropertyRef Id="WIX_ACCOUNT_LOCALSERVICE" />
+		<Property Id="ARPCONTACT" Value="opencode@microsoft.com" />
+		<Property Id="TARGETDIR" Secure="yes">
+			<RegistrySearch Id="FindInstallLocation" Root="HKLM" Key="Software\Microsoft\Windows\CurrentVersion\Uninstall\[WIX_UPGRADE_DETECTED]" Name="InstallLocation" Type="raw" Win64="yes" />
+		</Property>
+		
+		<!-- Define the Product features and installation steps -->
+		<Feature Id="ProductFeature" Title="XDP for Windows Installer" ConfigurableDirectory="TARGETDIR" Display="expand" Level="1" Absent="disallow" AllowAdvertise="no" InstallDefault="local" TypicalDefault="install" >	
+			<ComponentGroupRef Id="xdp_driver" />
+			<ComponentGroupRef Id="xdpapi_dll" />
+		</Feature>
+		
+		<InstallExecuteSequence>
+			<!--Rollback sequence-->
+			<Custom Action="xdp_uninstall_rollback" Before="xdp_install">NOT Installed</Custom>
+
+			<!--Install sequence-->
+			<Custom Action="xdp_install" After="InstallFiles">NOT Installed</Custom>
+
+			<!--Uninstall sequence-->
+			<Custom Action="xdp_uninstall" After="InstallInitialize">REMOVE="ALL"</Custom>
+		</InstallExecuteSequence>
+
+		<!-- Define the UI style & behavior -->
+		<UIRef Id="WixUI_Minimal" />
+		<WixVariable Id="WixUILicenseRtf" Value="$(var.SolutionDir)LICENSE.rtf" />
+		<InstallUISequence/>
+		<UI/>
+	</Product>
+
+	<!-- Define installation directories -->
+	<Fragment>
+		<Directory Id="TARGETDIR" Name="SourceDir">
+			<Directory Id="System64Folder" Name="System64Folder">
+				<Directory Id="dir_drivers" Name="drivers"/>
+			</Directory>
+		</Directory>
+	</Fragment>
+
+	<!-- Define the product components -->
+	<Fragment>
+		<!-- XDP driver components -->
+		<ComponentGroup Id="xdp_driver" Directory="dir_drivers">
+			<Component Id="xdp_setup" Guid="{0D991B56-5C01-4E57-AB33-E81696B73D55}">
+				<File Id="xdp_setup_ps1" Source="scripts\xdp-setup.ps1" KeyPath="yes"/>
+			</Component>
+			<Component Id="xdp.inf" Guid="{70A53BC2-8D56-4D53-B8FA-F07EC3D4783A}">
+				<File Id="xdp.inf" Name="xdp.inf" Source="$(var.TargetDir)xdp.inf" KeyPath="yes"/>
+			</Component>
+			<Component Id="xdp.sys" Guid="{5E1B9729-E58D-4A4D-A845-FB6D2EE0C498}">
+				<File Id="xdp.sys" Name="xdp.sys" Source="$(var.TargetDir)xdp.sys" KeyPath="yes" />
+			</Component>
+			<Component Id="xdp.pdb" Guid="{BC5C7A2C-613B-4831-A721-1FDFC4129F12}">
+				<File Id="xdp.pdb" Name="xdp.pdb" Source="$(var.TargetDir)xdp.pdb" KeyPath="yes" />
+			</Component>
```

Comment:
I don't thnk we need to be installing the pdb, right?

Reply:
good question. for now i'd say we should omit it, but in the future symbols should perhaps be an optional component.

Reply:
Your call, removing them.